openIMIS/developer_tools/skeletons.py

```diff
@@ -0,0 +1,53 @@
+def get_skeleton_setup(module_name, author, author_email):
+    return \
+        F"import os\n" \
```

Comment:
It's better to use os.linesep instead if '\n' in skeletons.

Reply:
ok. good point